__tests__/app.test.js

```diff
@@ -103,7 +103,7 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
-describe("GET /api/articles", () => {
+describe.only("GET /api/articles", () => {
```

Comment:
Remember to remove .only before submitting PRs. You want to keep main a clean version that's not got any leftovers from your debugging